src/monitoring_service/handlers.py

```diff
@@ -349,7 +357,7 @@ def monitor_new_balance_proof_event_handler(event: Event, context: Context) -> N
         # Unfortunately, parity does the gas estimation on the current block
         # instead of the next one, so we have to wait for the first allowed
         # block to be finished to send the transaction successfully on parity.
-        trigger_block = BlockNumber(channel.closing_block + channel.settle_timeout + 1)
+        trigger_block_timestamp = (channel.closing_block + 1) * 15
```

Comment:
The trigger block for the event should be the block right after the closing block plus the timeout.

When it comes to block numbers, that's easy to estimate. But now we need the timestamp of a block in the future. The best estimation we have for that block is to consider the 15 seconds/block time average. (And you probably noticed the missing settle timeout, that's because by the time of this change we still didn't have access to the token network timeout, so it will be included in the next commit).

Reply:
I don't think this is correct. It should be just the timestamp of the closing block plus the settle timeout. Because that is  what the contract is validating when you call the settle function. Correct? There is nothing about new blocks and block time involved here anymore.

Reply:
The comment above this explains that this is something very particular to parity